great-mystery-great-whore/original/129-chapter-128.adoc

```diff
@@ -1327,213 +1400,278 @@ but when I had been there a little while.
 I had given me what I was to declare; and ever since I have remained,
 not knowing today what I was to do tomorrow.
 
-Col. Brigs.
+[.discourse-part]
+_Col. Brigs._
 What was the promise thou hadst given?
 
-Jam.
+[.discourse-part]
+_Jam._
 That God would be with me; which promise I find made good every day.
 
-Col. Brigs.
+[.discourse-part]
+_Col. Brigs._
 I never heard such a call as this is, in our time.
 
-Jam.
+[.discourse-part]
+_Jam._
 I believe thee.
 
-Just. Pears.
+[.discourse-part]
+_Just. Pears._
 Is Christ in thee?
 
-Jam.
+[.discourse-part]
+_Jam._
 I witness him in me; and if I should deny him before men,
 he would deny me before my Father which is in heaven.
 
-Just. Pears.
+[.discourse-part]
+_Just. Pears._
 Spiritual, you mean.
 
-Jam.
+[.discourse-part]
+_Jam._
 Yea, spiritual.
 
-Just. Pears.
+[.discourse-part]
+_Just. Pears._
 By faith, or how?
 
-Jam.
+[.discourse-part]
+_Jam._
 By faith.
 
-Just. Pears.
+[.discourse-part]
+_Just. Pears._
 What difference then between the ministers and you?
 
-Jam.
+[.discourse-part]
+_Jam._
 The ministers affirm Christ to be in heaven with a carnal body,
 but I with a spiritual body.
 
-Just. Pears.
+[.discourse-part]
+_Just. Pears._
 Which of the ministers say Christ is in heaven with a carnal body?
 
-Jam.
+[.discourse-part]
+_Jam._
 The minister so called of Kirkby Steven.
 
+[.small-break]
+'''
+
```

Comment:
how about instead of the `[.small-break]` here we make the paragraph below `Priest Higginson stood up...` an `[.offset]`

Reply:
I went back and forth with [.small-break] and [.offset] 😄